docs/concepts/core-workflow-concepts.md

```diff
@@ -6,47 +6,45 @@ This document explains the core concepts that are needed to understand the workf
 
 ## Workflow
 
-A workflow is the definition of a process. It is made up of states and transitions between those states. The following diagram shows a simple workflow representing the lifetime of an item in a catalogue:
+A workflow is the definition of a process. It is made up of states and transitions between those states. The following diagram shows a  workflow representing a simple expense claim process:
```

Comment:
Double space between "a" and "workflow". (I think markdown goes out of its way to honour double spaces, right? So in HTML rendering it'll emit a non-collapsible space.)